### DISCUSSION TITLE
v1.0.2.4

### DISCUSSION THREAD

**Discussion by PeterSuh-Q3** [PeterSuh-Q3/tinycore-redpill#35] [Q&A] [answered]
<hr /><em>This discussion was created from the release <a href='https://github.com/PeterSuh-Q3/tinycore-redpill/releases/tag/v1.0.2.4'>v1.0.2.4</a>.</em>

**Comment by iziyou** [comment, marked as answer]
3622xs+ 7.1， does not recognize MLX5
